src/runtime/analytics-service.js

```diff
@@ -195,7 +195,7 @@ export class AnalyticsService {
    * @param {!../proto/api_messages.AnalyticsEvent} event
    * @return {!AnalyticsRequest}
    */
-  createLogRequest_(event) {
+  createLogRequest(event) {
```

Comment:
Just so you are aware, I am modifying this function to take the new ClientEvent object for events manager in a different pull request.  This is just FYI - I'm not sure who will beat who to final commit but you may need to resolve it on merge.  You can resolve this conversation (plus the other ones once you add the comments I asked for).

Reply:
Got it. Thanks for the heads up.